src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    static final Color[] colorValues = Color.values();
```

Comment:
Avoid using static methods or fields. Instead, you can move the initialization of `colorValues` inside the `getRandomColor` method.